src/test/util.js

```diff
@@ -30,18 +30,19 @@ import fs from 'fs-extra';
 
 import {exec} from 'child_process';
 import type {Test} from 'tape';
-
-import tmp from 'tmp';
+import uuid from 'uuid/v4';
 
 import {ThriftFileConverter} from '../main/convert';
 
+
 export const flowResultTest = (
   files: {[string]: string},
   testFn: (Function, FlowResult) => void,
   suffix: string = 'XXX',
   withsource: boolean = true
 ) => (t: Test) => {
-  const root = tmp.dirSync().name;
+  const root = path.resolve('test-output/', uuid());
+  fs.mkdirSync(root);
```

Comment:
Why are all these I/O ops synchronous? Since the exec is async and the function isn't done til testFn is called, why not make these all async (or promisify)

Reply:
sync or async doesn't really matter in this case since tests are run in series